registrations/management/commands/report_pmtct_risks.py

```diff
@@ -103,8 +104,14 @@ def add_to_result(risk, reg):
                     validated=True)
 
                 for registration in registrations:
+                    mom_dob = registration["data"].get("mom_dob")
+                    if not mom_dob:
+                        identity = self.get_identity(
+                            ids_client, registration["registrant_id"])
+                        mom_dob = identity["details"].get("mom_dob")
```

Comment:
Yeah, we probably don't need to update the mother's risk status if she's opted out.